<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OduncAlmaIslemleri</class>
 <widget class="QDialog" name="OduncAlmaIslemleri">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="uye_tablo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>321</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tüm Üyeler</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="kitap_tablo">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>40</y>
     <width>331</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tüm Kitaplar</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="odunc_tablo">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>330</y>
     <width>331</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>310</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ödünç Alınan Kitaplar Listesi</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="uye_no">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="kitap_no">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>370</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Üye No</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Kitap No</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ödünç Alma Tarihi</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="date_odunc">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>400</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_odunc_alma">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>450</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Ödünç Al</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
